conans/pylint_plugin.py

```diff
@@ -55,3 +55,34 @@ def _python_requires_member():
 
 
 astroid.register_module_extender(astroid.MANAGER, "conans", _python_requires_member)
+
+
+class ConanDeprecatedImportsChecker(BaseChecker):
+    """
+    Check "from conans*" imports which disappears in Conan 2.x. Only "from conan*" is valid
+    """
+
+    __implements__ = IRawChecker
+
+    deprecated_imports_pattern = re.compile(r"(from|import)\s+conans[\.|\s].*")
+    name = "conan_deprecated_imports"
+    msgs = {
+        "W9000": (
```

Comment:
@memsharded I put it as `"W" == "warning"`, should I change it by `"E" == "error"`

Reply:
Yes, mark it as error